core/src/main/scala/cats/syntax/functor.scala

```diff
@@ -1,4 +1,34 @@
 package cats
 package syntax
 
+import cats.evidence.<~<
+
 trait FunctorSyntax extends Functor.ToFunctorOps
+
+private[syntax] trait FunctorSyntaxBinCompat0 {
+  implicit final def catsSyntaxFunctorOps0[F[_], A](fa: F[A]): FunctorOps0[F, A] =
+    new FunctorOps0[F, A](fa)
+}
+
+final class FunctorOps0[F[_], A](private val fa: F[A]) extends AnyVal {
+
+  /**
+   * Un-zips an `F[(A, B)]` consisting of element pairs or Tuple2 into two separate F's tupled.
+   *
+   * NOTE: Check for effect duplication, possibly memoize before
+   *
+   * {{{
+   * scala> import cats.Id
+   * scala> import cats.syntax.functor._
+   *
+   * scala> (5: Id[Int]).map(i => (i, i)).unzip == ((5, 5))
+   * res0: Boolean = true
+   * }}}
+   *
+   */
+  def unzip[X, Y](implicit F: Functor[F], ev: A <~< (X, Y)): (F[X], F[Y]) = {
+    val fxy = F.map(fa)(ev)
+
+    F.unzip(fxy)
```

Comment:
Is there a reason not to write this as `F.unzip(F.map(fa)(ev))`? It's straightforward enough that I'm not sure the intermediate `val` helps much.

Reply:
@travisbrown Yeah I was thinking of doing that but I thought fxy would be clearer somehow. Will make the change to a one liner tomorrow.